torch/csrc/jit/passes/quantization/insert_observers.cpp

```diff
@@ -374,28 +382,75 @@ class InsertObserversHelper {
   void fillPassThroughValueMap(const std::shared_ptr<Graph>& graph);
 
   const ModuleQConfigMap& module_qconfig_map_;
+
   // Values we want to delay observation, used to delay the observation for
   // values in the middle of the ops that are supposed to be fused, e.g.
   // the output value of conv in the conv - relu pattern
   // the key is the intermediate output, e.g. output of conv
   // the value is the value we want to observe, e.g. output of relu
+  //
+  // example, assuming we want to delay conv-relu:
+  //   %x1 = conv(%x0)
+  //   %x2 = relu(%x1)
+  //
+  // delay_observation_map_ = {
+  //   %x1: %x2,
+  // }
   std::unordered_map<Value*, Value*> delay_observation_map_;
+
   std::unordered_set<Graph*> visited_graph_of_observer_map_;
+
+  // example:
+  //   %x1 = conv(%x0)
+  //
+  // observer_for_value_ = {
+  //   %x0: FooObserverModule,
+  //   %x1: BarObserverModule,
+  // }
   std::unordered_map<Value*, Module> observer_for_value_;
```

Comment:
not sure if example is helpful for this one, I think you can say this is the map from a value to the observer module configured by user.

Reply:
agreed, it's probably not that helpful to people who work on this code full time.  It's targeted more to newcomers / people who need to sometimes touch this code.

Reply:
I mean the other examples look good, but this one is probably not as helpful as the others